README.md

```diff
@@ -175,3 +175,37 @@ split:
     - git fetch --prune --unshallow || git fetch --prune
     - gitsplit --ref "${CI_COMMIT_REF_NAME}"
 ```
+
+# Sample with Github Actions
+
+I recommend to not use [the checkout action](https://github.com/actions/checkout) because it will checkout only the last
+commit for performance purpose. In this example we clone the whole repository. Indeed, if you need to split only the
+last changes and you don't need to split tags, you can use it.
+
+Also we do not use the GH Action auto-generated token because it doesn't have the required permissions for gitsplit to
```

Comment:
I'm confused, isn't `secrets.GH_TOKEN` the default token?
Would be great to provide a link to the personal page that to generate a new token.
ie. https://github.com/settings/tokens/new?scopes=repo&description=git-split

Reply:
It's the same name, but I overwrite it, probably confusing, I'll change the secret name :+1: and I'll add a link to generate the token.